.github/workflows/ci.yml

```
@@ -0,0 +1,205 @@
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: ['18', '20']
+        exclude:
+          # Reduce matrix size - only test Node 20 on Ubuntu
+          - os: windows-latest
+            node-version: '20'
+          - os: macos-latest
+            node-version: '20'
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)